src/lib/bootloader/grub_install.rb

```diff
@@ -42,7 +42,8 @@ def execute(devices: [], secure_boot: false, trusted_boot: false, update_nvram:
         Yast::Execute.on_target(cmd)
         # workaround for arm on SLE15 SP2 (bsc#1167015)
         # run grub2-install also non-removable if efi is there
-        if (Yast::Arch.aarch64 || Yast::Arch.arm) && Systeminfo.writable_efivars?
+        if (Yast::Arch.aarch64 || Yast::Arch.arm || Yast::Arch.riscv64) &&
```

Comment:
so this workaround is also needed on risc? To be honest I am not much happy about it as it is just workaround that is a bit strange.

Reply:
I don't know for sure, the approach taken here is to make it like aarch64, as the booting path is quite similar.

Reply:
basically that workaround is needed as some arm expect removable installation for efi and does not work without removable, so we install it to both locations. Not sure about risc as I am not familiar with that arch ( well, even with arm I am not so much familiar and depends mainly on experts ).

Reply:
can we do this in a followup?

Reply:
yes, fine for me.